dev-guide.md

```diff
@@ -12,6 +12,7 @@ To run Small Board in Docker, you will need:
 To run Small Board manually, you will need:
 
 - Git
+- Python
```

Comment:
Should we include a minimum version? Or at least say Python 3?

Reply:
here i don't think it actually matters? you just need it to install pipenv, and i think most methods of installing pipenv actually pull in python if you don't have it already. i just included it here for completeness